docs/content/about-iota/iota-architecture/epochs.mdx

```diff
@@ -9,42 +9,49 @@ import {questions} from '../../../site/static/json/about-iota/iota-architecture/
 
 ## Epoch
 
-In IOTA, an epoch is a set period, around 24 hours, during which the
-IOTA [validator](../tokenomics/validators-staking.mdx) set and their stakes remain constant. This stable period allows
-validators to handle transactions smoothly without unexpected changes.
+In IOTA, an epoch is a fixed period (typically around 24 hours) during which the [validator](../tokenomics/validators-staking.mdx) set and their stakes remain constant. This stable period ensures that [transactions](../../developer/iota-101/transactions/transactions.mdx) are processed smoothly without unexpected validator changes.
 
-Epoch values are part of the transaction data on the IOTA network, but regular users don't need to worry about them.
-Users only need to know about epochs if they are dealing with
-expiring [transactions](../../developer/iota-101/transactions/transactions.mdx), which must be completed before a
-specific epoch ends.
+## Epoch Numbering & Transaction Data
 
-## Reconfiguration
+Epochs are sequentially numbered, and each transaction on the IOTA network includes an epoch identifier. While regular users generally do not need to worry about epochs, they are relevant when dealing with expiring transactions, which must be completed before a specific epoch ends.
 
-Reconfiguration is an important process that happens at the end of each epoch to prepare the network for the next one.
-It includes these key steps:
+## Consensus & Checkpoints
 
-### 1. Finalizing Transactions and Checkpoints
+During each epoch, the Mysticeti consensus protocol operates to finalize transactions. Validators reach agreement by collecting transaction effect certificates, ensuring a consistent transaction order. A key part of this process is the checkpoint, which is a snapshot of finalized transactions at the end of an epoch. The checkpoint includes:
 
-The network agrees on the final set of transactions and checkpoints for the current epoch, ensuring all validators have
-the same information. This is the only time the network is fully synchronized, known as a _synchronous moment_, which is
-essential for consistency.
+- A finalized set of transactions
```

Comment:
Is it a "finalized set of transactions" or "set of finalized transactions"? The latter seems more appropriate.